.travis.yml

```diff
@@ -50,9 +50,8 @@ install:
   - sudo make install
 script:
   - echo "build successful"
-  - echo "cd ~/ngxpagespeed"
-  - echo "sudo ./test/run_tests.sh 8050 8051 $HOME/mod_pagespeed/src $HOME/nginx-branches-default/objs selfsigned.modpagespeed.com"
-  - echo "sudo ./test/run_tests.sh 8050 8051 $HOME/mod_pagespeed $HOME/nginx-branches-default/objs/nginx selfsigned.modpagespeed.com"
+  - cd ~/ngxpagespeed
+  - sudo ./test/run_tests.sh $HOME/mod_pagespeed $HOME/nginx-branches-default/objs/nginx
```

Comment:
I don't follow the update.py comment. If that's breaking all our Travis builds, we should fix it.

I think we should (1) fix that and (2) do the mps submodule thing before "uncorking" travis.